src/plugins/poster/poster.js

```diff
@@ -6,14 +6,18 @@ import UIContainerPlugin from 'base/ui_container_plugin'
 import Events from 'base/events'
 import Styler from 'base/styler'
 import template from 'base/template'
+import Playback from 'base/playback'
 import posterStyle from './public/poster.scss'
 import posterHTML from './public/poster.html'
 import playIcon from 'icons/01-play.svg'
 
 export default class PosterPlugin extends UIContainerPlugin {
   get name() { return 'poster' }
   get template() { return template(posterHTML) }
-  get shouldRender() { return this.container.playback.name !== 'html_img'}
+  get shouldRender() {
+    const showForNoOp = !!(this.options.poster && this.options.poster.showForNoOp)
```

Comment:
I think there's no need to explicitly check if `this.options.poster` is an object. 

LGTM